### PR TITLE
Consistent reset option

### PR DESCRIPTION
**Pull request by jonmcalder**
- Implement project reset option for all `createProject()` functions as discussed in #99 
- Re-order arguments for consistency

### CODE REVIEW

File: R/createBasicProject.R [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #'  "travis" means calling \code{usethis::use_travis()} and adding the Travis
 #'  badge to the README.
 #'  }
-#' @param reset whether to reset the project to current project
+#' @param reset Whether to reset the project to current project
```
Comment (maelle) [inline R/createBasicProject.R]: I am in awe of your attention to details :sparkles:

**Comment by maelle** [comment]
Thank you!

File: R/createBasicProject.R (hunk 1 above)
Comment (jonmcalder) [reply R/createBasicProject.R]: Thanks 😃

Although I find it can be both a blessing and a curse...

Comment (maelle) [reply R/createBasicProject.R]: well that's a blessing for this package :grin:

**Pull request merged**